backend/php_functions/create.php

```diff
@@ -33,7 +33,7 @@ function create_account($json_hash) {
 
 	array_push(
 		$queries,
-		"INSERT INTO Accounts(username,password) VALUES('$username', '$password')" 
+		"INSERT INTO Accounts(username,password) VALUES('$username', '$password') RETURNING account_id" 
```

Comment:
@markc331 can you check if this syntax is valid in MySQL? I tested in MariaDB, but I'm not sure if `RETURNING` clause is universal. If not, we may need to update instructions to specify that our project specifically requires MariaDB rather than generic MySQL/Percona/MariaDB...